E3Next/Settings/CharacterSettings.cs

```diff
@@ -455,7 +455,7 @@ public IniData CreateSettings(string fileName)
             newFile.Sections.AddSection("Assist Settings");
             section = newFile.Sections.GetSectionData("Assist Settings");
             section.Keys.AddKey("Assist Type (Melee/Ranged/Off)", "Melee");
-            section.Keys.AddKey("Melee Stick Point", "Back");
+            section.Keys.AddKey("Melee Stick Point", "Behind");
```

Comment:
to prevent this question from popping up in the future https://discord.com/channels/548643730714918913/1024676514119290901/1105623805075923085

Reply:
ty